backend/coins/coin/amount.go

```diff
@@ -76,6 +79,64 @@ func (amount Amount) BigInt() *big.Int {
 	return new(big.Int).Set(amount.n)
 }
 
+// FormattedAmountWithConversions and json tags.
+type FormattedAmountWithConversions struct {
```

Comment:
Maybe it would make sense to rename the frontend type from IAmount to IFormattedAmountWithConversions for consistency.

Reply:
Removed the "Formatted" bit to make it shorter, since it is not really relevant in the frontend.